### ISSUE TITLE
Long runs are broken

### ISSUE THREAD

**Issue by charleskawczynski** [CliMA/ClimaAtmos.jl#409]
Last night's long run broke with
```
ERROR: LoadError: SingularException(10)
  | Stacktrace:
  | [1] checknonsingular
  | @ /central/software/julia/1.7.1/share/julia/stdlib/v1.7/LinearAlgebra/src/factorization.jl:19 [inlined]
  | [2] lu!(A::Tridiagonal{Float32, Vector{Float32}}, pivot::RowMaximum; check::Bool)
  | @ LinearAlgebra /central/software/julia/1.7.1/share/julia/stdlib/v1.7/LinearAlgebra/src/lu.jl:563
  | [3] lu! (repeats 2 times)
  | @ /central/software/julia/1.7.1/share/julia/stdlib/v1.7/LinearAlgebra/src/lu.jl:496 [inlined]
```
The last working long run was Thursday, May 5, 12:05am.

Looking at the merge history and (mse table changes), the potential culprits are: #388, #398.

cc @dennisYatunin, @szy21, @simonbyrne.

Thoughts one which one was more likely the issue? Or what we can do to fix this?

**Comment by szy21** [comment]
It's probably the third order upwinding and maybe the Jacobian. Could we try to use the central difference in the long run first? @dennisYatunin added an option for that I think.

**Comment by charleskawczynski** [comment]
Closed by #413, thanks @szy21!

**Comment by szy21** [comment]
That’s great! It would still be good to figure out why it’s broken with third order upwinding though, especially because that’s what we are going to use. Maybe it requires a smaller dt? We can ask Paul or Tapio. Could you try adding a long run with third order upwinding and dt=300s? If it takes too long to run we can change t_end to 800 days (I think it crashed much earlier than this).
